packages/web-components/fast-components/src/toolbar/toolbar.definition.ts

```diff
@@ -0,0 +1,37 @@
+import type { WebComponentDefinition } from "@microsoft/fast-tooling/dist/esm/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+import { Orientation } from "@microsoft/fast-web-utilities";
+
+export const fastToolbarDefinition: WebComponentDefinition = {
```

Comment:
@janechu made an update to definition files the other day, I think this may need to update